scripts/data_deletion/delete_training_data_utils.js

```diff
@@ -207,7 +205,7 @@ var filterFamilyMembers = function(personsList, logdir) {
 
 // Find which facilities (if any) this person is a contact for.
 var isContactFor = function(db, personId) {
-  return db.query('medic-client/places_by_contact', {key: [personId], include_docs: true})
```

Comment:
The only reason anything is in medic-client is if it's needed by the front end browser code. It's possible this was once needed but now it's more appropriate on server side only.